### PR TITLE
Marketplace : apply feedback

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

File: server/sonar-web/src/main/js/store/marketplace/actions.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+interface SetEditionStatusAction {
+  type: 'SET_EDITION_STATUS';
+  status: EditionStatus;
+}
+
+export type Action = LoadEditionsAction | SetEditionsAction | SetEditionStatusAction;
+
+export function loadEditions(loading: boolean = true): LoadEditionsAction {
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/store/marketplace/actions.ts]: MINOR Type boolean trivially inferred from a boolean literal, remove type annotation [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3257)

File: server/sonar-web/src/main/js/app/components/nav/settings/SettingsNav.tsx [hunk 2]
```diff
@@ -77,7 +80,7 @@ export default class SettingsNav extends React.PureComponent<Props> {
   };
 
   render() {
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/app/components/nav/settings/SettingsNav.tsx]: CRITICAL The function render has a cyclomatic complexity of 14 which is higher than the threshold of 10 [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1541)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 3]
```diff
@@ -2192,7 +2192,7 @@ system.hide_intermediate_versions=Hide intermediate versions
 system.how_to_upgrade=How to upgrade?
 system.is_restarting=Server is restarting. This page will be automatically refreshed.
 system.latest_version=Latest Version
-system.latest_lts_version=Latest LTS Version
+system.lts_version=LTS Version
```
Comment (stas-vilchik) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: I found a typo: "Please **resstart** the server to activate your new features."

File: server/sonar-web/src/main/js/app/components/nav/settings/SettingsEditionsNotifContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  preventRestart: boolean;
+}
+
+interface DispatchToProps {
+  setEditionStatus: (editionStatus: EditionStatus) => void;
+}
+
+const mapStateToProps = (state: any): StateToProps => ({
+  editions: getMarketplaceEditions(state),
+  preventRestart: getMarketplaceState(state).readOnly || !getAppState(state).standalone
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/app/components/nav/settings/SettingsEditionsNotifContainer.tsx]: fix this

**Comment by gregaubert** [comment]
@stas-vilchik I fixed your comments and added the spinner while checking the license.

**Pull request merged**